catalog/USGS/catalog.jsonnet

```diff
@@ -63,5 +63,7 @@ local self_url = base_url + base_filename;
     ee.link.child_collection('USGS_WBD_2017_HUC08', base_url),
     ee.link.child_collection('USGS_WBD_2017_HUC10', base_url),
     ee.link.child_collection('USGS_WBD_2017_HUC12', base_url),
+
+    ee.link.child_collection('USGS_NLCD_RELEASES_2021_REL_TCC_v2021-4', base_url),
```

Comment:
don't change catalog.jsonnet yet, please, as long as gee:skip_indexing is set. Right now we have this two-part process when the inactive dataset definition has to be checked in first, after which we generate images internally and make the dataset active. We'll streamline this process soon.